packages/kmod-6.1-nvidia-r570/kmod-6.1-nvidia-r570.spec

```diff
@@ -515,6 +532,22 @@ popd
 # systemd units
 %{_cross_unitdir}/nvidia-persistenced.service
 
+# ICD / vendor descriptors
```

Comment:
For consistency with the spec, could you please move `%dir` directives where all the others are for `%files tesla`?

Reply:
Done, I also alpha sorted this, but happy to just append the new dirs if preferred.